src/modules/discord/commands/factionStatus.ts

```diff
@@ -110,11 +110,16 @@ export class FactionStatus {
                                                     let influence = responseObject[0].history[0].influence;
                                                     let pendingStatesArray = responseObject[0].history[0].pending_states;
                                                     let recoveringStatesArray = responseObject[0].history[0].recovering_states;
-
                                                     let factionDetail = "";
+                                                    let updatedAt = new Date(responseObject[0].updated_at);
+                                                    let currentTime = new Date();
+                                                    if (currentTime < updatedAt) {
```

Comment:
What is this check for? Current time is always going to be lesser than updated one, right?